daily_tests/sclorg_sanity_tests.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python3
+import os
+import sys
+import requests
+import yaml
+import urllib3
+import smtplib
+
+from email.mime.application import MIMEApplication
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from tempfile import TemporaryDirectory
+from pathlib import Path
+from typing import Dict
+from shutil import rmtree
+from git import Repo
+from git.exc import GitCommandError
+
+SCLORG_REPOS = [
+    # Format is 'repo_name' and 'package_name'
+    ("s2i-base-container", "s2i-base"),
```

Comment:
I there any reason why the base container has the `s2i` prefix when others do not have it?
```suggestion
    ("s2i-base-container", "base"),
```